Flank/flank
# refactor: Enable properties file in flank scripts

Enable properties file to be used when testing `flank-scripts`.
Now you can use `flank-debug.properties` to replace some crucial settings (like repo). Custom properties are used only during local runtime (CI and tests use the defaults)

### Test plan
1. Build project `./gradlew build`
1. Observe all tests finish without an error
1. `/common/flank-debug.properties` should be created
1. Run `java -jar ./flank-scripts/build/libs/flank-scripts.jar shell firebase checkForSdkUpdate --github-token=[your gh token] --zenhub-token=""`
1. Default settings are used, there should be similar out to: 
    ```
    ** Last workflow run:
     name: Update dependencies
     last run: 2021-01-06T10:05:04Z
     url: https://github.com/Flank/flank/actions/runs/465789230
    ```
1. In `flank-debug.properties` uncomment `repo.flank` and change to any random value
1. Run again `java -jar...`
1. Output should contain `** No workflow run found for update_dependencies_and_client.yml`
1. Comment/undo `repo.flank` and change value for `sdk-check.workflow-filename`
1. Run `java -jar....`
1. There should be output with `** No workflow run found for [name you changed]`

Please, feel free to play with other scritps
